src/components/DashboardNoTeam/ConfirmTeamCreation/ConfirmTeamCreation.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { useMutation, gql } from '@apollo/client';
+import { Box, Text, Heading, Button, HStack, VStack } from '@chakra-ui/react';
+
+const ConfirmTeamCreation = props => {
```

Comment:
Same comment related to destructuring props as `src/components/DashboardNoTeam/InitialPage/InitalPage.js`